### DISCUSSION TITLE
A call for ideas to increase financial contributions

### DISCUSSION THREAD

**Discussion by fasttime** [eslint/eslint#18957] [General]
We are reaching out to the GitHub community to help us explore new ways to increase the number and value of donations through the ESLint website, social media, and other channels.

As an open source project, ESLint thrives thanks to the contributions of a community of maintainers, developers, and users like you. Community support, both through code contributions and financial donations, is essential for keeping the project growing and sustainable.

During the past two years, our recurring costs have increased, which is a positive signal because we pay contributors for all non-trivial contributions. To put it another way: we’ve had more contributors this year than in the previous years. But these growing costs have not been matched by an increase in financial donations. In order to stay sustainable, we need to increase the project income from around $10,000 per month to at least $15,000 per month. To achieve this goal, we are planning to launch a “call for sponsorship” campaign that will leverage social media channels and include a blog post and adjustments to our website. But we are all developers and pretty new to this kind of campaign, which is why we need concrete recommendations to put our plans into practice.

## Why your input matters
Open source is all about collaboration, and just as we work together to build great software, we can also collaborate to ensure its financial sustainability. Your insights can help us:
- Make it easier for people to donate
- Highlight the impact of financial contributions
- Find creative ways to engage and reward sponsors
- Reach new potential sponsors within and outside the ESLint community

**Do you have an idea on how to promote the above objectives, or do you even have any experience in raising funds for an open source project? If so, we would love to hear what you have to say. Leave your comment below to let us know!**

**Comment by Splines** [comment]
I don't have any experience in this, but I just wanted to point to the [Blender Development Fund](https://fund.blender.org/) as a source for inspiration. They retrieve 164,030€ monthly by 4497 individuals and 36 corporate institutions, just to see the scale at which they operate.

They kind of gamify this process a bit by displaying latest activity (`Someone donated $10`) and using the concept of ["Levels"](https://fund.blender.org/corporate-memberships/) for different annual fees (at corporate memberships). And of course, the big sponsors are showcased on the landing page of that fund.

For end-users like me, they also make it really easy to donate by showing an unintrusive `Donate to Blender` page after having downloaded the software:

![image](https://github.com/user-attachments/assets/801f0493-3681-4ad0-94f6-41bb0429bc34)

Some more remarks from my personal donation experience
- I expect PayPal to be available as payment option. I'd also like to pay in my local currency (for me EUR €).
- I think transparency is very important: what is the money used for? How much money do you receive by donations? Maybe even showcase this somehow like Blender does. They provide many information, neatly organized on their [About page](https://fund.blender.org/about/).
- Corporate sponsors might want to know about tax benefits when donating.
- `npm fund`: In my dev experience, after installing a package and seeing many lines running through in the terminal (including warnings), I usually ignore the output of `npm fund`, it's just too much noise for me. If I really like a tool, I will rather donate when browsing through the tools website/documentation.

**Reply by Splines** [reply]
Ah and additionally, I think one shouldn't underestimate *familiarness / customer intimacy*: the Blender team creates this by occasionally uploading videos from their studio in the Netherlands. Like [this video celebrating their 30th birthday](https://youtu.be/3fdTIV85l98). I mean it's just fun to see there's actual people behind the software ;) Developers even show how they develop Blender in C++ in YouTube videos sometimes.

**Reply by nzakas** [reply]
Thanks for sharing this. I think we cover a lot of your points on our donate page: https://eslint.org/donate.

What do you think?
